pkg/sql/plan2/function/operator/isnotnull.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package operator
+
+import (
+	"errors"
+
+	"github.com/matrixorigin/matrixone/pkg/container/nulls"
+	"github.com/matrixorigin/matrixone/pkg/container/vector"
+	"github.com/matrixorigin/matrixone/pkg/vm/process"
+)
+
+func IsNotNull[T DataValue](vectors []*vector.Vector, proc *process.Process) (*vector.Vector, error) {
+	lv := vectors[0]
+	rv := vectors[1]
+	l := int64(1)
+
+	if !rv.IsScalarNull() {
```

Comment:
so when the input is  a const vector , you wanna return error...
why not a const vector of type boolean, value false, please state your reasons

Reply:
what's the difference between left, right input vectors

Reply:
will fix it by use 1 input vector